### PR TITLE
Fix "A PART OF" Code style for non-public, non-static fields #8387

### PR DESCRIPTION
**Pull request by Arnold2381**
## Purpose / Description
Change the formatting issues according to #8370 

## Fixes
Fixes some of #8387 

## Approach
Followed #8370, added those changes ran gradlew lintRelease to generate a report of all the issues and solve errors accordingly. 

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented on your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by david-allison-1 — COMMENTED** [review]
Forgot to hit submit here

File: AnkiDroid/src/main/java/com/ichi2/libanki/importer/python/CsvReaderIterator.java [hunk 1]
```diff
@@ -35,27 +35,27 @@
 import static com.ichi2.libanki.importer.python.CsvReaderIterator.State.*;
 
 public class CsvReaderIterator implements Iterator<List<String>> {
```
[the remark this answers is not in the document]
Comment (Arnold2381) [reply AnkiDroid/src/main/java/com/ichi2/libanki/importer/python/CsvReaderIterator.java:37]: Ok sure!

**Review by david-allison-1 — APPROVED** [review]
Looks good to me!

Maintainers: Squash & merge - there was a fixup commit in the middle

**Review by mikehardy — APPROVED** [review]
so....much....code haha

**Pull request merged**